src/components/user/unavailability/unavailability.edgedb.repository.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@nestjs/common';
+import { Range } from 'edgedb';
+import { ID, PublicOf } from '~/common';
+import { e, RepoFor } from '~/core/edgedb';
+import {
+  CreateUnavailability,
+  Unavailability,
+  UpdateUnavailability,
+} from './dto';
+import { UnavailabilityRepository } from './unavailability.repository';
+
+@Injectable()
+export class UnavailabilityEdgeDBRepository
```

Comment:
@CarsonF - I'm not sure why this is showing that the `create` method is only returning an ID.  I would expect it to return the entire option since the `this.hydrate` is in there?

Reply:
Replied in other thread.